__tests__/app.test.js

```diff
@@ -258,3 +258,77 @@ describe("POST /api/reviews/:review_id/comments", () => {
       });
   });
 });
+describe("PATCH /api/reviews/:review_id", () => {
+  it("should return 201 with the updated review object", () => {
+    return request(app)
+      .patch("/api/reviews/2")
+      .send({ inc_votes: 2 })
+      .expect(201)
```

Comment:
A status code of 201 is not appropriate for a patch request, instead simply have a 200 status code is fine for this type of request.